src/assets/calendar.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="26" height="24" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I suspect we might not have permission to include all these icons in an open source repo, unless we include some kind of copyright/license disclaimer would be sufficient, since these are probably all trademarked images.

It might be better to serve these from an S3 bucket URL, and just have the paths here. Another option would be to just include generic logos here, and (if desired) replace them with more specific logos in a private, instance specific "skinny" repo.

Reply:
It looks like we can use their icons to link to their services as long as we follow their [guidelines](https://about.google/brand-resource-center/guidance/apis/). I'm happy to use S3 or a skinny, but I think all instances would likely be identical in this part.